### ISSUE TITLE
Bug report v1.10.573

### ISSUE THREAD

**Issue by darvinya** [AnySoftKeyboard/AnySoftKeyboard#1711]
The Hungarian language dictionary is still bad in this version because it offers stupid things that are not Hungarian.
For example, you can see in the screenshot that "Es" is offered instead of "És", which is not Hungarian
You can barely use your own input and save dictionary or store only 1 piece of a newly saved word.
Display is defective, visible on screen.

![Screenshot_20190429-135810](https://user-images.githubusercontent.com/46437370/56894664-de584700-6a86-11e9-9041-908114cdc89c.png)

Does not work emoji window automatic closure.
App settings menus are bad, eg: Automatically saving words Do not save any words. 

In short, it is also a big bug set.
Only a user can be deterred.

Much better keyboard 2013.12 month version because it works on dictionary, etc.
There is only one flaw that Android 9 does not have a good pop-up window.

Android OS version: Android 9
Device manufacturer and model: Xiaomi Mi A2
List of installed add-ons (like languages, or themes): Hungarian, Dark Theme
